Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1577,9 +1577,20 @@ namespace Babylon
 
         // TODO: support other fill modes
         uint64_t fillModeState = 0; //indexed tri list
-        if (fillMode == 2)
+        switch (fillMode)
```

Comment:
I think we should try to be more consistent in passing enums. Sometimes we pass bgfx enums. Sometimes we pass Babylon enums. Should we just pick one and stick with it?

Reply:
Agree. Opening an issue for that. https://github.com/BabylonJS/BabylonNative/issues/406